src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NuGetManagedTemplatePackage.cs

```diff
@@ -234,24 +169,24 @@ public bool IsLocalPackage
 
         public IReadOnlyDictionary<string, string> GetDetails()
         {
-            Dictionary<string, string> details = new Dictionary<string, string>();
-            if (!string.IsNullOrWhiteSpace(Author))
-            {
-                details[AuthorKey] = Author!;
-            }
-            if (!string.IsNullOrWhiteSpace(Owners))
-            {
-                details[OwnersKey] = Owners!;
-            }
-            if (!string.IsNullOrWhiteSpace(Trusted))
-            {
-                details[TrustedKey] = Trusted!;
-            }
-            if (!string.IsNullOrWhiteSpace(NuGetSource))
+            var details = new Dictionary<string, string>();
+
+            details.TryToAdd(AuthorKey, Author!, InsertionCondition);
+            details.TryToAdd(OwnersKey, Owners!, InsertionCondition);
+            details.TryToAdd(TrustedKey, Trusted!, InsertionCondition);
+            details.TryToAdd(NuGetSourceKey, NuGetSource!, InsertionCondition);
+
+            return details;
+        }
+
+        private bool InsertionCondition(string entry) => !string.IsNullOrEmpty(entry);
```

Comment:
previously the condition was `string.IsNullOrWhiteSpace(entry)`

Reply:
the condition was inverted. 
![image](https://user-images.githubusercontent.com/95473390/231440287-20fd7ed3-70fa-4854-b213-c2b198067a0d.png)